cases/models.py

```diff
@@ -1,35 +1,70 @@
 from django.db import models
 
+class Country(models.Model):
+        name = models.CharField(max_length=255, unique=True)
+        slug = models.SlugField(unique=True)
+        time_created = models.DateTimeField(auto_now_add=True)
+        last_updated = models.DateTimeField(auto_now=True)
+
+        class Meta:
+            ordering = ['name']
+
+        def __str__(self):
+            return f'{self.name}'
 
 class Visual(models.Model):
-    country = models.CharField(max_length=255)
-    case = models.TextField()
-    death = models.TextField()
-    recovery = models.TextField()
-    #
-    time_created = models.DateTimeField(auto_now_add=True)
-    last_updated = models.DateTimeField(auto_now=True)
-
-    class Meta:
-        ordering = ['country']
-
-    def __str__(self):
-        return f'{self.country} {self.case}'
-
-
-class KenyanCase(models.Model):
```

Comment:
How will we save the Kenyan cases?

Reply:
I am yet to Implement the Kenyan Cases part. I just commented it all out.

Reply:
Cool. However it's better not to comment out code that you have not replaced.